testData/actions/generation/generator/ModuleBlockClassGenerator/generateFile/ViewBlock.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
```

Comment:
With tests everything is ok. I have a question about the `declare(strict_types=1);` Should we added it to all PHP *.ft files and to the tests as well?

Reply:
It can be configured in the PHP header template.
Take a look at my configuration
![image](https://user-images.githubusercontent.com/20116393/80352605-c9221e00-887c-11ea-8a99-305b926aef59.png)

Reply:
Aha, got it. So this parameter is in a user responsibility are. As for me, this is ok too 👍